ambry-router/src/main/java/com.github.ambry.router/RouterUtils.java

```diff
@@ -76,6 +76,7 @@ static boolean isSystemHealthError(Exception exception) {
         case BadInputChannel:
         case BlobDeleted:
         case BlobExpired:
+        case BlobDoesNotExist:
```

Comment:
Well, if a new error code is added to the api, then the implementation either has to deal with that error code or ignore it - based on how the implementation does that, its test will have to be changed accordingly. If the main code deals with the error, it should get added here as well.

Made a change here to deal with all error codes in the test.